src/ui/components/ErrorBoundary.tsx

```diff
@@ -24,20 +25,22 @@ export default function ErrorBoundary({ children }: { children: ReactNode }) {
       return dispatch(setUnexpectedError(ReplayUpdatedError, true));
     }
 
-    if (error.name === "CommandError" || isDevelopment()) {
+    if (error.name === "CommandError") {
       return;
     }
 
-    setUnexpectedError({
-      message: "Unexpected error",
-      content: "An unexpected error occurred. Please refresh the page.",
-      action: "refresh",
-    });
+    dispatch(
+      setUnexpectedError({
+        message: "Unexpected error",
+        content: "Something went wrong, but you can refresh the page to try again.",
+        action: "refresh",
+      })
+    );
   };
 
   return (
     <Sentry.ErrorBoundary onError={onError}>
-      {unexpectedError ? <BlankViewportWrapper /> : children}
+      {unexpectedError ? <Error /> : children}
```

Comment:
Would this look weird if it rendered inside of a nested component?

Reply:
Hmm. So it looks like this component is only used in two places:

- `_app.tsx`
- `Editor/Footer.tsx`

Everything else uses the `bvaughn/components/ErrorBoundary` component

Let me swap the old editor footer to use that component too, so this only gets used around the main app

Reply:
Sounds good!